.gitignore

```diff
@@ -402,3 +402,8 @@
 /utils/wxrc/wxrc_vc[789].sln
 
 /3rdparty/webview2
+
+/.vs
+/.vscode
+/build_cmake
```

Comment:
A nitpick, but I'd prefer to use `build/cmake/build` or something like this (it's unfortunate that `build` is redundant, but I'd still like to keep everything CMake-related under `build/cmake`).

Reply:
I've changed it to `build/cmake/output`